src/macaron/config/defaults.ini

```diff
@@ -233,39 +231,41 @@ wrapper_files =
 [builder.gradle.ci.build]
 github_actions = actions/setup-java
 travis_ci =
-    jdk
-    ./gradlew
+    gradle
 circle_ci =
-    ./gradlew
+    gradle
 gitlab_ci =
-    ./gradlew
+    gradle
 jenkins =
-    ./gradlew
```

Comment:
I wonder why we don't have `gradle` for jenkins ?

Reply:
Not sure why `./gradle` was not used, but searching for these as keywords is not needed anymore.